code/src/index.css

```diff
@@ -1,13 +1,181 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap');
```

Comment:
If you would like to do this in styled componenets instead you could do it with global components